Clients/GUI/Scripts/SignUper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GUI.GUIModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using DTO;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace GUI.Scripts
+{
+    public class SignUpper
+    {
+        public static HttpStatusCode SignUp(SignUpViewModel input)
+        {
+            var output = new UserEmailPassword(input.Email, input.Password);
+
+            var request = (HttpWebRequest)WebRequest.Create("https://localhost:5011/user/create");
```

Comment:
Is there a way to avoid hardcoding the url?

Reply:
What is hardcoding mean?